SEED-platform/seed
# Fix/bsync meterreading import

#### What's this PR do?
- makes MeterReading's `conversion_factor` a required field. Any existing meter readings which have a NULL value and have kbtu for units will have conversion factor set to 1.0. If the migration finds a record that has a NULL value and it's units are not kbtu, it raises an exception.
- Fixes example file (buildingsync_v2_0_bricr-workflow.xml) by making ResourceUse IDs unique
- Fixes merging of properties (Actions -> Merge Selected) with BuildingSync files
  - Merging previously assumed all meter readings had converion_factor field set, but imported building files did not. Fixed by setting it to 1.0 for all meter readings (assumed kBtu)
  - fixes bug where meter-scenario information was dropped when merging properties
#### How should this be manually tested?
https://drive.google.com/drive/folders/1w9d1mX6MxMaZi0fsTP3HX_WItP2G6WCk
1. Upload properties along with meter data (Monthly Metrics_Sample Data Request Response - 4 Properties  &  example-pm-monthly-meter-usage).
2. Upload a BuildingSync file (buildingsync_v2_0_bricr-workflow.xml)
2. From the Properties listing, select a property and the building sync property and click Actions -> Merge Selected. It should be successful, and you should verify that on the meters page:
  - meter data from both files is present.
  - scenario information for the building sync meters is retained (click the filter by, and select 'scenario')
#### What are the relevant tickets?
#2131
